src/Interpreters/GlobalSubqueriesVisitor.h

```diff
@@ -135,7 +135,8 @@ class GlobalSubqueriesMatcher
                 ast = database_and_table_name;
 
             external_tables[external_table_name] = external_storage_holder;
-            subqueries_for_sets[external_table_name].source = QueryPipeline::getPipe(interpreter->execute().pipeline);
+            subqueries_for_sets[external_table_name].source = std::make_unique<QueryPlan>();
+            interpreter->buildQueryPlan(*subqueries_for_sets[external_table_name].source);
```

Comment:
It's the second place where we create `QueryPlan` just to `buildQueryPlan()` it at the next line. Would it better to make a combined function `interpreter->createQueryPlan()`?

Reply:
I will probably change an interface of `buildQueryPlan` later.